### ISSUE TITLE
TypeError: can't multiply sequence by non-int of type 'Forward'

### ISSUE THREAD

**Issue by Masterchiefm** [lightdock/lightdock#87]
**Describe the bug**
I encounter this ERROR when I use " LightDock setup". I was running your [simple docking tutorial](https://lightdock.org/tutorials/0.9.3/simple_docking), and when I run the follow command
 ```lightdock3_setup.py 2UUY_rec.pdb 2UUY_lig.pdb --noxt --noh --now -anm``` I got this output:

```
ightdock3_setup] INFO: Saving processed structure to PDB file...
[lightdock3_setup] INFO: Done.
[lightdock3_setup] INFO: Saving processed structure to PDB file...
[lightdock3_setup] INFO: Done.
[lightdock3_setup] INFO: Calculating ANM for receptor molecule...
Traceback (most recent call last):
  File "/home/chief/miniconda3/envs/py38/bin/lightdock3_setup.py", line 93, in <module>
    calculate_anm(receptor, args.anm_rec, args.anm_rec_rmsd, args.anm_seed, DEFAULT_REC_NM_FILE)
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/lightdock/prep/simulation.py", line 136, in calculate_anm
    modes = calculate_nmodes(parsed_lightdock_structure, num_nmodes, rmsd, seed, structure)
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/lightdock/structure/nm.py", line 42, in calculate_nmodes
    num_atoms_prody = prody_molecule.select("protein or nucleic").numAtoms()
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/prody/atomic/atomic.py", line 232, in select
    return SELECT.select(self, selstr, **kwargs)
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/prody/atomic/select.py", line 895, in select
    indices = self.getIndices(atoms, selstr, **kwargs)
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/prody/atomic/select.py", line 953, in getIndices
    torf = self.getBoolArray(atoms, selstr, **kwargs)
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/prody/atomic/select.py", line 1004, in getBoolArray
    parser = self._getParser(selstr)
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/prody/atomic/select.py", line 1103, in _getParser
    parser.enablePackrat()
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/pyparsing/util.py", line 265, in _inner
    return fn(*args, **kwargs)
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/pyparsing/core.py", line 1132, in enable_packrat
    ParserElement.packrat_cache = _FifoCache(cache_size_limit)  # type: ignore[assignment]
  File "/home/chief/miniconda3/envs/py38/lib/python3.7/site-packages/pyparsing/util.py", line 105, in __init__
    keyring = [object()] * size
TypeError: can't multiply sequence by non-int of type 'Forward'

```

I tried turn to python=3.8 or 3.7, but the error still there.

**Env (please complete the following information):**
 - OS:  Ubuntu 22.04
 - Python 3.8

**Additional context**

**Comment by alchemistcai** [comment]
Here is the same issue and its troubleshooting #76 @Masterchiefm

**Comment by Masterchiefm** [comment]
Someone have the same issue and it can be solved by ```pip install pyparsing==3.1.1```

**Comment by Masterchiefm** [comment]
> Here is the same issue and its troubleshooting #76 @Masterchiefm

Thanks, it works!
